### DISCUSSION TITLE
theme.extend.colors overwrites defaults

### DISCUSSION THREAD

**Discussion by Anyass3** [windicss/windicss#141] [Q&A] [answered]
I think the  `theme.extend.colors` should be extending the color defaults but it overwrites it

for example:
```js
// tailwind.config.js
module.exports = {
  theme: {
    extend: {
      colors: {
        red: {
          custom: '#ea1851',
        },
      },
    },
  },
};
```
This overwrites all the default red color shades for `custom` with windicss

I tried same thing with tailwindcss postcss compiler and it was working

https://tailwindcss.com/docs/customizing-colors#extending-the-defaults

**Comment by hannoeru** [comment, marked as answer]
Fixed it, will be released in next version.

**Reply by Anyass3** [reply]
great thanks
